<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerador De Numeros</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Original</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0" colspan="6">
         <widget class="QLabel" name="lblBuscarOriginal">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: #dddddd;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: black;
  vertical-align: middle;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="4">
         <widget class="QPushButton" name="btnGerarSemente">
          <property name="styleSheet">
           <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
          </property>
          <property name="text">
           <string>Gerar Semente!</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="btnBuscarOriginal">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
          </property>
          <property name="text">
           <string>Buscar!</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="btnGerarNumeros">
          <property name="styleSheet">
           <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
          </property>
          <property name="text">
           <string>Gerar Numeros!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="6">
         <widget class="QTextEdit" name="textOriginal">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 30px;
background-color: #dddddd;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLineEdit" name="lineQuantidade">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: rgb(255, 255, 255);
  display: block;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: #black;
  vertical-align: middle;
  background-color: #ffffff;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Semente:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QLineEdit" name="lineBuscarOriginal">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: rgb(255, 255, 255);
  display: block;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: #black;
  vertical-align: middle;
  background-color: #ffffff;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineSemente">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: rgb(255, 255, 255);
  display: block;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: #black;
  vertical-align: middle;
  background-color: #ffffff;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Quantidade:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Selection Sort</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnSelectionSortC">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
            </property>
            <property name="text">
             <string>Crescente</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectionSortD">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
            </property>
            <property name="text">
             <string>Decrescente</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTextEdit" name="textSelectionSort">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 30px;
background-color: #dddddd;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineBuscaSSR">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: rgb(255, 255, 255);
  display: block;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: #black;
  vertical-align: middle;
  background-color: #ffffff;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnBuscaSSR">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
          </property>
          <property name="text">
           <string>Buscar Recursiva!</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="lblBuscaSSR">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: #dddddd;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: black;
  vertical-align: middle;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineBuscaSSI">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: rgb(255, 255, 255);
  display: block;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: #black;
  vertical-align: middle;
  background-color: #ffffff;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btnBuscaSSI">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">  padding: 6px 12px;
  margin-bottom: 0;
  font-size: 20px;
  font-weight: normal;
  line-height: 1.428571429;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 4px;
  background-color: #428bca;
  border-color: #357ebd;</string>
          </property>
          <property name="text">
           <string>Buscar Interativa!</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="lblBuscaSSI">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
background-color: #dddddd;
  width: 100%;
  height: 34px;
  line-height: 1.428571429;
  color: black;
  vertical-align: middle;
  background-image: none;
  border: 1px solid #cccccc;
  border-radius: 4px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
